js/plugins/laboratory/src/components/LabsEditModal.js

```diff
@@ -0,0 +1,270 @@
+            lymphocytes: this.lymphocytesRef.current.value,
+            neutrophils: this.neutrophilsRef.current.value,
+            eosinophils: this.eosinophilsRef.current.value,
+            basophils: this.basophilsRef.current.value,
+            monocytes: this.monocytesRef.current.value,
+        };
+
+        const { labsData } = this.props;
+        this.props.handleSave(Object.assign({}, labsData || {}, newLabsData));
+    };
+
+    getValue = text => {
+        if (!text || !text.split) {
+            return text || '';
+        }
+
+        const [value] = text.split(' ');
+        return value;
+    };
+
+    render() {
+        const { date, leukocytes, hemoglobin, platelets, lymphocytes, neutrophils, eosinophils, basophils, monocytes } =
+            this.props.labsData || {};
+
+        return (
+            <Dialog open={this.state.isOpen} onClose={() => this.toggleModal(false)}>
+                <DialogTitle>
+                    <Box display="flex" alignItems="center">
+                        <Box flexGrow={1}>
+                            <Typography variant="h3">
+                                {t('ehr', this.props.labsData ? 'Update Labs' : 'Add New Lab Results')}
+                            </Typography>
+                        </Box>
+                        <Box>
+                            <IconButton onClick={() => this.toggleModal(false)}>
+                                <CloseIcon />
+                            </IconButton>
+                        </Box>
+                    </Box>
+                </DialogTitle>
+                <DialogContent>
+                    <form>
+                        <Grid container>
+                            <FormControl>
+                                <KailonaDateTimePicker
+                                    inputRef={this.dateRef}
+                                    id="date"
+                                    ariaLabel={t('ehr', 'Select Date/Time')}
+                                    defaultValue={date ? moment(date) : null}
+                                    disableFuture={true}
+                                />
+                            </FormControl>
+                        </Grid>
+                        <Grid container alignItems="center">
+                            <GridColumn className="left-column" item>
+                                <FormControl>
+                                    <KailonaTextField
+                                        inputRef={this.leukocytesRef}
+                                        id="leukocytes"
+                                        className="kailona-MuiTextField"
+                                        label={t('ehr', 'Leukocytes')}
+                                        defaultValue={this.getValue(leukocytes)}
+                                        InputProps={{
+                                            endAdornment: <InputAdornment>10&#94;3/&micro;l</InputAdornment>,
```

Comment:
It is a bit hard to see the value since the unit looks like the value. Could you gray out the unit a bit in the table?

<img width="257" alt="Screen Shot 2021-07-25 at 14 08 41" src="https://user-images.githubusercontent.com/8215016/126897001-8da91a04-14c6-4cb8-99e6-f4d5c125c2e2.png">